# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/webidl/WakeLock.webidl
# Commit: 231b0b4efd16
# Full Hash: 231b0b4efd16880f3273859e7dc97b4cfe789c21
# Author: Vincent Hilla <vhilla@mozilla.com>
# Date: 2023-12-06 05:18:37
# Regressor Bug: 1589554
# File Overlap Count: 1
# Description:
#   Bug 1589554 - Part 2: Implement Screen Wake Lock API. r=dom-core,edgar
#   
#   Depends on D189508
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D189509
# ==============================================================================

diff -r f50ff29a1a64 -r 231b0b4efd16 dom/webidl/WakeLock.webidl
--- a/dom/webidl/WakeLock.webidl	Tue Dec 05 23:58:06 2023 +0000
+++ b/dom/webidl/WakeLock.webidl	Tue Dec 05 23:58:06 2023 +0000
@@ -9,6 +9,7 @@
 
 [SecureContext, Exposed=(Window)]
 interface WakeLock {
+  [Throws]
   Promise<WakeLockSentinel> request(optional WakeLockType type = "screen");
 };
 